.circleci/config.yml

```diff
@@ -25,9 +25,18 @@ build-node-base: &node-base
     - run:
         name: Versions
         command: yarn versions
+    - &restore-yarn-cache
```

Comment:
Looks good!
Just a suggestion of the approach I've been using for these lately:
WDYT about writing it like that:
```yaml
 -  restore_cache: &yarn
        key: yarn-{{ .Environment.CIRCLE_JOB }}-{{ checksum "package.json" }}
        paths:
          - ./node_modules
          - ./yarn.lock
    - run:
        name: Install dependencies	       
        command: yarn install	    
    - save_cache: *yarn
```

And then you can use it like so:

```
- restore_cache: *yarn
- save_cache: *yarn
```

Reply:
The reason I usually prefer the syntax used in the PR is that if the object in the array has multiple keys, it would still work. This avoids having to remember and possibly extract all those keys individually if they should always be the same.

Take for example the `docker-base` anchors which all contain an `image` key, and in some cases also an `environment` key.

Reply:
Makes sense 👍 thanks for explanation